tools/ci/templates/jobs-mrwebrtc-release-build.yaml

```diff
@@ -0,0 +1,190 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
What's the difference between jobs-cpp.yaml and this file?

Reply:
Looking at them side by side it seems that jobs-cpp uses the NuGets for webrtc-uwp, while this builds from source. Is there a reason to have these as separate jobs sequences, or can we merge them (or at least extract common steps in templates)

Reply:
`jobs-cpp` builds for CI. We should probably try to merge some parts where possible, but I wanted to get that PR out so at least we have something working, and we can improve after.